libfreerdp/core/input.c

```diff
@@ -479,6 +539,9 @@ BOOL input_recv(rdpInput* input, wStream* s)
 {
 	UINT16 i, numberEvents;
 
+	if (!input || !s)
```

Comment:
I think that stream NULL pointer check is not necessary as this function is called from the transport layer where stream creation is already checked and ensured.